app/models/pregnancy.rb

```diff
@@ -50,4 +51,22 @@ class Pregnancy
   #   end
   #   false
   # end
+  
+  private
+  
+  def updating_pledge_sent?
+    pledge_sent == true
+  end
+
+  def check_other_validations?
```

Comment:
What are your thoughts about renaming this method? Usually having `?` at the end of a method indicates it either returns true or false, and this isn't really doing this in the current (very good and useful!) implementation. Instead, it's confirming that necessary information is saved so it can mark the pledge sent. Thoughts?